### PR TITLE
Adds greeting using fetch statement.

### PR DESCRIPTION
**Pull request by meagancheese**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -91,3 +91,8 @@ function changeImageBorders(color){
     document.images.item(i).style.border = "10px solid " + color;
   }
 }
+
+function sayHi() {
+  fetch('/data').then(response => response.text()).then(
+    document.getElementById('greeting-container').innerText = 'Hello Meagan!');
```
Comment (acmcarther) [inline portfolio/src/main/webapp/script.js:97]: Did you mean to use response.text() here?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -91,3 +91,8 @@ function changeImageBorders(color){
     document.images.item(i).style.border = "10px solid " + color;
   }
 }
+
+function sayHi() {
+  fetch('/data').then(response => response.text()).then(
```
Comment (acmcarther) [inline portfolio/src/main/webapp/script.js]: Nit: My preference would be to do this work (response.text()) when setting the element text, not as a separate step here. As is, it looks like response.text() returns a promise (though javascript supports both promise returning and value returning functions in `then()`).

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```
Comment (acmcarther) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: This seems like a separate change that should be in a separate PR, but I don't feel strongly abou this, just keep an eye out in the future.

**Review by acmcarther — APPROVED** [review]
I strongly suggest using the description to explain the change. It's not super important in these smaller changes, but getting into the practice early pays dividends in the future when you can't remember what a change was and don't want to read the whole diff.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (acmcarther) [reply portfolio/src/main/webapp/script.js:97]: Also, this doesn't look syntactically correct... you probably meant `() =>` or `(text) =>`

Comment (meagancheese) [reply portfolio/src/main/webapp/script.js:97]: I saw it in the example code, is it not necessary here? I wasn't sure what Promises had to be satisfied before I sent the greeting.

**Pull request merged**